actors/builtin/miner/miner_test.go

```diff
@@ -484,25 +494,21 @@ func TestExtendSectorExpiration(t *testing.T) {
 		actor.extendSector(rt, sector, extension, params)
 
 		// assert sector expiration is set to the new value
-		st = getState(rt)
+		st := getState(rt)
 		sector, found, err := st.GetSector(rt.AdtStore(), sector.Info.SectorNumber)
 		require.NoError(t, err)
 		require.True(t, found)
 		assert.Equal(t, newExpiration, sector.Info.Expiration)
 
-		// assert that an expiration exists at the target epoch
-		expirations, err := st.GetSectorExpirations(rt.AdtStore(), newExpiration)
-		require.NoError(t, err)
-		exists, err := expirations.IsSet(uint64(sector.Info.SectorNumber))
-		require.NoError(t, err)
-		assert.True(t, exists)
-
-		// assert that the expiration has been removed from the old epoch
-		expirations, err = st.GetSectorExpirations(rt.AdtStore(), expiration)
-		require.NoError(t, err)
-		exists, err = expirations.IsSet(uint64(sector.Info.SectorNumber))
-		require.NoError(t, err)
-		assert.False(t, exists)
+		// assert the only expiration is for the new target epoch
+		err = st.ForEachSectorExpiration(rt.AdtStore(), func(expiry abi.ChainEpoch, sectors *abi.BitField) error {
```

Comment:
This will be a false negative if the code removes the expiration and fails to add it at the new epoch. Adding a length check outside the iteration would cover this, but I still think the previous version is more clear in its intent.

Reply:
The motivation for the change was that the `expiration` var was no longer in scope, and this was attempting to assert the complete state of the expirations. You're right about the false negative, though. I can get it from the old sector info and will change back.